dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -252,8 +256,17 @@ public abstract class Entry<S extends Entry, T extends Version> implements Compa
 
     @Column
     @ApiModelProperty(value = "The Digital Object Identifier (DOI) representing all of the versions of your workflow", position = 14)
+    @Schema(description = "The Digital Object Identifier (DOI) representing all of the versions of your workflow", deprecated = true)
+    @Deprecated(since = "1.16")
     private String conceptDoi;
 
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
A little wary of eager, but am I right in assuming there will be at most three of these since this is for concept DOIs?

Reply:
Yes, it should be a max of 3, but I can make it lazy if it's preferred since we're re-designing the DOI stuff anyway

Reply:
I think I can live with three since it's a constant value unlike past instances where we had a value grow based on the number of versions. 
I think we can document the max of 3  under `@Schema` 
Apparently, you can also do `@Size(max=3)`

Reply:
> I think the GitHub-Zenodo integration could in theory create multiple DOIs for the same entry, as the integration is configured at the Zendo account level. So if Kathy and Denis both enable Zenodo integration for the GitHub dockstore/dockstore repo, my understanding is that a GitHub release in that repo would trigger the creation of 2 DOIs.

Tested it on zenodo.org (the sandbox wasn't syncing my organizations for some reason) and Zenodo actually doesn't allow multiple users to enable the integration if it's already enabled by someone else in your organization. See the tooltip over the `ON` button in the screenshot below:

![image](https://github.com/dockstore/dockstore/assets/25287123/cede9444-54d7-4f1a-baf2-75c962d566d4)

So I think it's safe to assume that the GitHub-Zenodo integration doesn't allow for the creation of multiple DOIs for a single release of a repository. The integration is only ever enabled once for each repository

Reply:
> Could we put the concept dois in a `Set` rather than a `Map`?

Hmm, we could but I put it in a map so it's easy to look up DOIs based on the creator

Reply:
Thanks for looking into that. It makes me wonder how they handle ownership of the DOI -- what happens if I enable DOI integration on this repo, then retire and I'm dropped from the GitHub dockstore organization. Am I still the owner of the DOI? But beyond the scope of this PR, just idle thoughts...

Reply:
On further reflection, I think it will still be possible for multiple DOIs to be issued, but they won't probably won't follow the same pattern that GitHub/Zenodo integration does, so we probably won't pick them up, and I think we can keep this as is.